bears/documentation/DocumentationStyleBear.py

```diff
@@ -49,10 +53,13 @@ def run(self, filename, file, language: str, docstyle: str='default'):
 
             # 1 empty line shall follow main description (except it's empty or
             # no annotations follow).
-            if main_description.desc.strip() != '':
+            if allow_missing_func_desc and main_description.desc.strip() != '':
                 main_description = main_description._replace(
-                    desc='\n' + main_description.desc.strip() + '\n' *
-                         (1 if len(parsed) == 1 else 2))
+                    desc='\n')
+            elif main_description.desc.strip() != '':
+                main_description = main_description._replace(
+                   desc='\n' + main_description.desc.strip() + '\n' *
+                   (1 if len(parsed) == 1 else 2))
```

Comment:
@jayvdb 
> If you re-indent this, the git should retain the attribution of this block and add your own, resulting in a cleaner/simpler diff.

uhh what do you mean by this? I didn't get you.
Yeah regarding why the commit's are seperate cause im finishing a bear, and totally hijacking over SanketDG's PR :stuck_out_tongue: